### ISSUE TITLE
Failed to execute 'attachShadow'

### ISSUE THREAD

**Issue by dimaqq** [textlint/editor#59]
Failed with:
`Failed to execute 'attachShadow' on 'Element': Shadow root cannot be created on a host which already hosts a shadow tree.`

Source: chrome-extension://gfhlfpnpjokocfohicmfbgjneiipfeil/scripts/pageScript.js in HTMLElement.connectedCallback at line 112:1910

Chrome version:93.0.4577

**Comment by dimaqq** [comment]
The stack trace is weird, it seems that that happens when user code calls `this.appendChild( elem );` in random jQuery

**Comment by azu** [comment]
Fmm, Can you provide reproduce website?

> Shadow root cannot be created on a host which already hosts a shadow tree.

That web site may have used Web Component.

**Comment by dimaqq** [comment]
No, sorry... The website is oldschool (jQuery), it wouldn't know about web components.
This came from automated reporting from customer system.
If someone manages to reproduce this, I'll update here.
Also, I can't be sure about other extensions that user's browser may have had installed.
